scripts/zones/Southern_San_dOria/npcs/Kipopo.lua

```diff
@@ -29,11 +52,22 @@ function onTrigger(player,npc)
 end;
 
 function onEventUpdate(player,csid,option)
+
 end;
 
 function onEventFinish(player,csid,option)
-    if (csid == 651 and option == 1) then
+    if (csid == 912) then
+        player:giveKeyItem(dsp.ki.REPAIRED_HANDBAG)
+        player:setVar("sayItWithAHandbagCS", 4)
+    elseif (csid == 910) then
+        player:delKeyItem(dsp.ki.TORN_PATCHES_OF_LEATHER)
+        player:setVar("sayItWithAHandbagCS", 3)
+        player:needToZone(true)
+        player:tradeComplete()
```

Comment:
tradeConfirm and then it will only take items asked for

Reply:
changed to player:confirmTrade()
Thanks for the back and forth in discord, scripting is still new to me